### DISCUSSION TITLE
Welcome to pForth Discussions!

### DISCUSSION THREAD

**Discussion by philburk** [philburk/pforth#98] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by znmeb** [comment]
I haven't done much with pForth yet but I've been following it for years. I'm glad to see it's active again!
